### DISCUSSION TITLE
Simple playlist(s) player mode à la foobar2000 (no library)

### DISCUSSION THREAD

**Discussion by jesus2099** [quodlibet/quodlibet#4221] [Q&A] [answered]
Hello,

Coming from Windows, [I am looking for a foobar2000 replacement](https://community.metabrainz.org/t/looking-for-a-foobar2000-equivalent-on-linux/483198?u=jesus2099) on Linux and quodlibet seems nice and is even multi-platform!

It does not have global hotkeys but it looks I can use `xbindkeys` + `quodlibet --playpause` etc.

But there is some things I could not manage.
I just want to double-click a file and it should play, or right click play a folder.

It seems the only way I could do this basic task was `quodlibet --run --play-file myfile` by command line (!).
The default action is not wiping the playlist and playing the given file(s)/folder(s).
But even doing so, I don't see the current playlist (I tried all the Navigation menu items, I have v4.2.1 on Raspberry Pi). And then it wants to play the ... library.

I could not play audio CD tracks either on my other PC with CD drive (Debian 10), is it normal?

In fact I don't need the library concept, I just have files in folders that I want to arrange and play in various playlist tabs, like in foobar2000. 

## Main Question ##

Is a simple free playlist(s) player mode possible?

**Comment by declension** [comment]
> In fact I don't need the library concept, I just have files in folders that I want to arrange and play in various playlist tabs, like in foobar2000.

OK, that's exactly what the _File System_ browser is for then.

> I just want to double-click a file and it should play, or right click play a folder.

Yes that's a bit harder than it perhaps should be in QL, but people often associate in their system file explorer (Nautilus etc) with a similar command line as you use there. See #1949 and a couple of others 

>I could not play audio CD tracks either on my other PC with CD drive (Debian 10), is it normal?

Yes that's unsupported, see #163 ; you'd need to convert first.

**Comment by bassstorm** [comment, marked as answer]
> In fact I don't need the library concept, I just have files in folders that I want to arrange and play in various playlist tabs, like in foobar2000.

You better go with [DeadBeef](https://github.com/DeaDBeeF-Player/deadbeef) then.

**Reply by jesus2099** [reply]
Thank you, I was advised this but its package was not in the Debian apt catalogue, I don't know why...

**Reply by bassstorm** [reply]
The options are a) just run binary, or perhaps preferrably b) install a `.deb` package manually. Please refer to https://deadbeef.sourceforge.io/download.html. Should be a really good fit for your needs :-)

P.S. Not available in various distros by default because the author has very own vision on how to build app and deal with its dependencies, that often come in a conflict with how distros do their job.

**Reply by jesus2099** [reply]
I find it less clean and more difficult to keep track of installed programs, when it's not with apt.

I've been using [audacious](https://audacious-media-player.org/) for this reason, even if it is less complete than foobar2000.

I'm not listening much music on PC, anyway, mostly on my hi-fi CD player. So I stopped looking for a foobar2000 replacement, at the moment.

**Reply by bassstorm** [reply]
Makes sense. But just in case, I'm a long time user of foobar2000 on Linux. Combination of portable install and Wine works pretty much perfectly.

**Reply by jesus2099** [reply]
I did not think yet about sharing my concern for lack of apt package install: https://github.com/DeaDBeeF-Player/deadbeef/issues/2985
